src/app/p/[projectId]/components/PayForm.tsx

```diff
@@ -4,11 +4,17 @@ import { useAccount } from "wagmi";
 import { useNativeTokenSymbol } from "../hooks/useNativeTokenSymbol";
 import { useJBTerminalContext, useJbMultiTerminalPay } from "juice-sdk-react";
 import { NATIVE_TOKEN } from "juice-sdk-core";
+import DecimalsInput from "@/components/ui/decimalsInput";
+import { useState } from "react";
 
 export function PayForm({ projectId }: { projectId: bigint }) {
+  const [value, setValue] = useState("0.069");
```

Comment:
maybe just default this to `"0"` and set a placeholder. wdyt?

Reply:
Oh word. Fixed.